info.html

```diff
@@ -18,8 +18,57 @@
       <nav class="include" data-include="header">
       </nav>
       <div id="content" >
-                <!-- place your <div class="include" data-include=""></div> here and copy the html into the views folder. Name it something like "page"-"component" -->
-      </div>
+        <!-- place your <div class="include" data-include=""></div> here and copy the html into the views folder. Name it something like "page"-"component" -->
+        <div class="container">
+          <center><h1>Info</h1></center>
+            <i><h3>"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."</h3></i>
+          <hr>
+          <center><h2>Community leaders</h2></center>
+          <div class="col-sm-4 leader">
+            <center><img src="" alt="" class="community"></center>
+            <center><h2>Gwen Faraday</h2>
+            <p>Camp Site Leader</p></center>
+          </div>
+          <div class="col-sm-4 leader">
+            <center><img src="" alt="" class="community"></center>
+            <center><h2>Jared Wilcurt</h2>
+            <p>Co-Organizer</p></center>
+          </div>
+          <div class="col-sm-4 leader">
+            <center><img src="" alt="" class="community"></center>
+            <center><h2>Matthew Allbright</h2>
+            <p>Website Project Leader</p></center>
+          </div>
+        </div><!-- end community leaders -->
+        <div class="container">
+          <div class="row">
+            <div class="col-md-12">
+              <center><h1>FreeCodeCamp Code of Conduct</h1></center>
+            </div>
+            <div class="row c-o-c">
```

Comment:
Use `code-of-conduct` or `codeOfConduct` rather than shorthand so the meaning of the class name is more obvious to others.